airbyte-webapp/src/packages/cloud/views/auth/OAuthLogin/OAuthLogin.tsx

```diff
@@ -0,0 +1,107 @@
+import { useRef, useState } from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+import { useUnmount } from "react-use";
+import { Subscription } from "rxjs";
+
+import { Spinner } from "components";
+
+import { useExperiment } from "hooks/services/Experiment";
+import { useAuthService } from "packages/cloud/services/auth/AuthService";
+
+import githubLogo from "./assets/github-logo.svg";
+import googleLogo from "./assets/google-logo.svg";
```

Comment:
Let's discuss that maybe tomorrow on our tech time. I think we should try to go for imported assets whereever possible, since it will enable optimization on those as well as automatically embed them (leading to faster load times). Also I personally feel having them close by the actual component using them, would help keeping concerns together.